guides/common/modules/proc_acd_installation.adoc

```diff
@@ -45,6 +45,41 @@ endif::[]
 # {foreman-maintain} service restart
 ----
 
+[id="{context}_installation_{SmartProxyServer}"]
```

Comment:
@adahms would you know offhand if this heading styling will cause us problems long term?